lxd/device/gpu_physical.go

```diff
@@ -342,16 +527,82 @@ func (d *gpuPhysical) pciDeviceDriverOverrideIOMMU(pciDev pcidev.Device, driverO
 	return nil
 }
 
+func (d *gpuPhysical) stopCDIDevices(configDevices *cdi.ConfigDevices, runConf *deviceConfig.RunConfig) error {
```

Comment:
does configDevices need to be a pointer?

Reply:
missing doc block

Reply:
no need for a pointer actually. Fixed.